src/style.css

```diff
@@ -1 +1,87 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+:root {
+  --text-font: Avenir, sans-serif;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
```

Comment:
The selector '.html' seems to be a mistake. If you intended to style the `<html>` element, you should use the `html` selector without the dot.